### PR TITLE
Release v2.1.0

### PR DESCRIPTION
**Pull request by theacodes**
`git log v2.0.2..HEAD | grep 'Merge pull request ' | awk -F ' ' '{print $4 $5}' | sort` yields:
#488 - now() -> utcnow() for deviceinfo.
#503 - _JWTAccessCredentials.

### CODE REVIEW

**Comment by jonparrott** [comment]
@nathanielmanistaatgoogle I can create the tag and release, or you can do it if you want.

**Comment by nathanielmanistaatgoogle** [comment]
@jonparrott: please do so.

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +19,7 @@
 * Adding common `sign_blob()` to service account types and a
   `service_account_email` property. (#421)
 * Improving error message in P12 factory
-  `ServiceAccountCredentials.from_p12_keyfile` when pyOpenSSL is
+  `ServiceAccountCredentials.frogit gadm_p12_keyfile` when pyOpenSSL is
```
Comment (dhermes) [inline CHANGELOG.md:22]: @jonparrott You fell asleep on your keyboard? Or you just really wanted to frog it

tgmofa1

Comment (theacodes) [inline CHANGELOG.md:22]: Focus-stealing Chrome did this to me. I feel so betrayed. I'll fix.

**Pull request merged**